src/main/java/org/embl/mobie/lib/create/ImagesCreator.java

```diff
@@ -296,48 +296,47 @@ public void addOMEZarrImage( String uri,
                 imageDataFormat,
                 ThreadHelper.sharedQueue );
 
-        if ( ! isImageValid( imageData, projectCreator.getVoxelUnit() ) ) {
-            return;
-        }
-
         if ( ! is2D( imageData ) && projectCreator.getDataset( datasetName ).is2D() ) {
             throw new UnsupportedOperationException("Can't add a 3D image to a 2D dataset" );
         }
 
         // If an image of the same name is inside the project, delete it when overwrite=True.
-        File oldImageFile = new File(imagesDirectory, imageName + ".ome.zarr" );
-        if (oldImageFile.exists()) {
-            if (overwrite) {
-                IJ.log("Overwriting image " + imageName + " in dataset " + datasetName );
-                deleteImageFiles( datasetName, imageName );
-            } else {
-                throw new UnsupportedOperationException("An image called " + imageName + "already exists in the dataset " +
-                        datasetName + ", and overwrite is set to false" );
+        if ( addMethod.equals( ProjectCreator.AddMethod.Copy ) )
```

Comment:
Don't you want this for linking too? Say there's a local ome-zarr image stored in your MoBIE project named "test". If I now add a link to a remote image called "test" - would this still be an issue? I'd assume you only want one image per name.

Reply:
Yes, only one image per name.